src/apphosting/githubConnections.ts

```diff
@@ -90,13 +90,32 @@ const ADD_ACCOUNT_CHOICE = "@ADD_ACCOUNT";
 const MANAGE_INSTALLATION_CHOICE = "@MANAGE_INSTALLATION";
 
 /**
- * Prompts the user to link their backend to a GitHub repository.
+ * Prompts the user to create a GitHub connection.
  */
-export async function linkGitHubRepository(
+export async function createGitHubConnection(
   projectId: string,
   location: string,
-): Promise<devConnect.GitRepositoryLink> {
+  connectionId: string | null,
```

Comment:
Nit

```suggestion
  connectionId: string?,
```

Reply:
This doesn't work. 

src/apphosting/githubConnections.ts:98:17 - error TS8020: JSDoc types can only be used inside documentation comments.

98   connectionId: string?,
                   ~~~~~~~

Reply:
oops should be `connectionId?: string`, updated.

Reply:
done